### PR TITLE
[1LP][RFR] New performance requirements

### PR DESCRIPTION
**Pull request by otsuman**
PRT failures not related to changes

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 80c8786a5e82700f42f01d5718349e676ff19c0c:

`cfme/fixtures/version_info.py`:
- :bangbang: Line 153:5: **E265** *block comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/fixtures/version_info.py [hunk 1]
```diff
@@ -151,6 +150,8 @@ def generate_version_files():
     starttime = time.time()
     ssh_client = SSHClient()
     relative_path = os.path.relpath(str(results_path), str(os.getcwd()))
+    # avoid importing outside perf testing
```
Comment (mshriver) [inline cfme/fixtures/version_info.py]: :+1: thanks!

Comment (RonnyPfannschmidt) [reply cfme/fixtures/version_info.py]: this would always run?!

Comment (mshriver) [reply cfme/fixtures/version_info.py]: This fixture is only used by perf tests

Comment (RonnyPfannschmidt) [reply cfme/fixtures/version_info.py]: i see, thanks

**Pull request merged**